src/app/item-page/simple/metadata-representation-list/metadata-representation-list.component.ts

```diff
@@ -97,14 +98,17 @@ export class MetadataRepresentationListComponent extends AbstractIncrementalList
    * @param metadata  The list of all metadata values
    * @param page      The page to return representations for
    */
-  resolveMetadataRepresentations(metadata: MetadataValue[], page: number): Observable<MetadataRepresentation[]> {
+  resolveMetadataRepresentations(metadata: MetadataValue[], page: number): Observable<MetadataRepresentation[]|any[]> {
     return observableZip(
       ...metadata
         .slice((this.objects.length * this.incrementBy), (this.objects.length * this.incrementBy) + this.incrementBy)
         .map((metadatum: any) => Object.assign(new MetadataValue(), metadatum))
         .map((metadatum: MetadataValue) => {
-          if (this.metadataService.isVirtual(metadatum)) {
+          if (this.metadataService.isVirtual(metadatum) && !metadatum.authority.includes('http')) {
             return this.relationshipService.resolveMetadataRepresentation(metadatum, this.parentItem, this.itemType);
+          } else if (this.metadataService.isVirtual(metadatum) && metadatum.authority.includes('http')) {
+            // TODO: we could do authority virtual handling here?
```

Comment:
I'm not sure I understand the meaning of this TODO.  Is it necessary, or should we instead add details on why this is returning an empty result when `http` is included?

Reply:
i have reverted the changes to this file for now as I believe this is related to virtual authority metadata / other GND functionality, not pure geospatial data rendering, good spotting